rpc/jsonrpc/types/chainsvrresults.go

```diff
@@ -28,6 +28,9 @@ type DecodeScriptResult struct {
 
 // EstimateSmartFeeResult models the data returned from the estimatesmartfee
 // command.
+//
+// Deprecated: This will be removed in the next major version bump.
```

Comment:
Looks like you deprecated the wrong one.  Shouldn't this be the one in fees?

Reply:
DOH!

Reply:
Isn't this the right one?

Reply:
In chain.go:
`rpcclient/chain.go:1080:func (r *FutureEstimateSmartFeeResult) Receive() (float64, error)`
It uses a float.

Reply:
Yeah, for some reason I thought it was a duplicate in the fees package we were talking about, but this does raise the point that the output of `handleEstimateSmartFee` really should be this result and not just the float.  I asked @matheusd about it, who implemented this code, and he agreed that it really should be returned this result.

So, this PR should not deprecate the result.  Instead, we should, via a future PR and a major RPC API version bump, modify the handler to return the full result.